entities/capitalist_card.rb

```diff
@@ -0,0 +1,49 @@
+class CapitalistCard < BaseCard
+  def initialize
+    super
+    @balance = DEFAULT_BALANCE_CAPITALIST
+    @type = CARD_TYPES[1]
+  end
+
+  def number
+    @number ||= generate_card_number
+  end
```

Comment:
You have this method in all 3 derived classes (UsualCard, CapitalistCard, VirtualCard), don't you think it's a good reason to put it in the `BaseCard` class as a shared logic.
Move it to `BaseCard#initialize` this way calling of super will be justified, if you want to refer to number without `@` - add attr_reader

Reply:
Moved ```@number``` to ```BaseCard```